### PR TITLE
Fix issue forms attributes

### PR DESCRIPTION
**Pull request by anesabml**
## What do these changes do?

Remove render attribute from input fields and use description attribute instead of about.

## Are there changes in behavior for the user?

Non.

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names.
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` for example (588.bugfix)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -61,7 +61,6 @@ body:
   attributes:
     label: Python Version
     description: Attach your version of Python.
-    render: console
```
Comment (webknjaz) [inline .github/ISSUE_TEMPLATE/bug_report.yml]: Just use textarea instead. The pre-filled text suggests a multiline output of a command, not just one line of input.

Comment (anesabml) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Yes, I agree `textarea` will be better.

**Review by webknjaz — APPROVED** [review]

**Comment by webknjaz** [comment]
Great, it works! I encourage you to test the new forms and try to assess if anything needs to be improved and if everything is straightforward for the submitters.

**Comment by anesabml** [comment]
I just tested it out and I think It does look straightforward and it's much better than the regular markdown templates.

**Comment by webknjaz** [comment]
Alright, let's monitor messages for feedback from the submitters to see their perception.

**Pull request merged**